results/migrations/0018_add_partners_2012_data.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+import json
+from itertools import chain
+
+from django.core.files import File
+from django.db import migrations
+from django.utils.text import slugify
+
+from cpm_generic.migration_utils import (add_subpage, get_content_type,
+                                         get_image_model, remove_subpage)
+
+MIGRATION_DIR = os.path.dirname(__file__)
+DIR_0017 = os.path.join(MIGRATION_DIR, '0018_add_partners_2012_data')
+
+
+# TODO: coulds be abstracted
```

Comment:
After one passed sync with Stas (near september i believe) we discussed raw data and decided to make copy when migration making to avoid faild migration due to data changes by not programmers